src/components/manage/Form/Field.jsx

```diff
@@ -64,6 +64,25 @@ const getWidgetFromTaggedValues = (widgetOptions) =>
     ? config.widgets.widget[widgetOptions.frontendOptions.widget]
     : null;
 
+/**
+ * Get widget props from tagged values
+ * @param {object} widgetOptions
+ * @returns {string} Widget component.
+ *
+
+directives.widget(
+    "fieldname",
+    frontendOptions={
+        "widget": "specialwidget",
+        "widgetProps": {"prop1": "specialprop"}
+    })
+
+ */
+const getWidgetPropsFromTaggedValues = (widgetOptions) =>
```

Comment:
This could be documented at https://docs.voltocms.com/recipes/widget/ (I know you're asking help on this!)

Reply:
Done